Mathlib/RingTheory/Int/Basic.lean

```diff
@@ -30,30 +30,23 @@ unique factors
 namespace Int
 
 
-theorem gcd_eq_one_iff_coprime {a b : ℤ} : Int.gcd a b = 1 ↔ IsCoprime a b := by
-  constructor
-  · intro hg
-    obtain ⟨ua, -, ha⟩ := exists_unit_of_abs a
-    obtain ⟨ub, -, hb⟩ := exists_unit_of_abs b
-    use Nat.gcdA (Int.natAbs a) (Int.natAbs b) * ua, Nat.gcdB (Int.natAbs a) (Int.natAbs b) * ub
-    rw [mul_assoc, ← ha, mul_assoc, ← hb, mul_comm, mul_comm _ (Int.natAbs b : ℤ), ←
-      Nat.gcd_eq_gcd_ab, ← gcd_eq_natAbs, hg, Int.ofNat_one]
-  · rintro ⟨r, s, h⟩
-    by_contra hg
-    obtain ⟨p, ⟨hp, ha, hb⟩⟩ := Nat.Prime.not_coprime_iff_dvd.mp hg
-    apply Nat.Prime.not_dvd_one hp
-    rw [← natCast_dvd_natCast, Int.ofNat_one, ← h]
-    exact dvd_add ((natCast_dvd.mpr ha).mul_left _) ((natCast_dvd.mpr hb).mul_left _)
+@[deprecated (since := "2024-11-10")]
+alias ⟨gcd_eq_one_iff_coprime.mpr, gcd_eq_one_iff_coprime.mp⟩ := isCoprime_iff_gcd_eq_one
```

Comment:
I don't see why we'd need this

Reply:
I was hoping that this was smart enough to make the suggestion, `isCoprime_iff_gcd_eq_one.mp` for `gcd_eq_one_iff_coprime.mpr`, but it isn't. Is is a bit of a shame that we can't define the alias `gcd_eq_one_iff_coprime := isCoprime_iff_gcd_eq_one`.
I will remove those extra aliases